instance/tests/models/test_utils.py

```diff
@@ -685,53 +687,53 @@ def test_put_no_prefix(self):
         value = 'value'
         agent.put(key, value)
 
-        _, data = self.client.kv.get(key)
-        fetched_value = data['Value'].decode()
+        _, test_data = self.client.kv.get(key)
```

Comment:
@giovannicimolin, why are these changes needed and how are they related to this PR?

Reply:
The name of the variable was `data`, so I kept getting pylint errors because I was overriding `ddt.data`. That's why I've changed these names.

Reply:
@giovannicimolin, we usually just import `ddt` and use it as `ddt.data`, `ddt.unpack` :smiley:

Reply:
I've reverted these changes